tests/unit/test_legacy_api.py

```diff
@@ -1,57 +1,12 @@
 """Tests for Flake8's legacy API."""
 from __future__ import annotations
 
-import argparse
-import configparser
-import os.path
 from unittest import mock
 
 import pytest
 
 from flake8.api import legacy as api
 from flake8.formatting import base as formatter
-from flake8.options import config
-
-
-def test_get_style_guide():
```

Comment:
I deleted this unit test since it's just a copy paste of the implementation -- there's an integration test which covers the same pathway